src/components/ListDetail/ListDetail.module.css

```diff
@@ -158,14 +159,19 @@
     width: 9px; /* 1 */
     background: var(--eds-theme-color-border-neutral-subtle); /* 2 */
     border-radius: 50%; /* 3 */
+
+    /* stylelint-disable-next-line max-nesting-depth */
+    .list-detail__item.eds-is-active & {
+      background: var(--eds-theme-color-text-neutral-default); /* 4 */
+    }
   }
 
   /**
   * List detail item variants
   * 1) Remove the bullet since the icon will take its place
   */
   li[class*='list-detail__item--'] & {
-    /* stylelint-disable-next-line max-nesting-depth */
```

Comment:
Why was `max-nesting-depth` removed from this line?

Reply:
Because I was getting two linting errors on the same line; one for max-nesting-depth and then a second one for the order of the selectors. I tried re-ordering a few different ways and wasn't able to find a combination that the linter liked so I threw in the towel on the entire line.

Reply:
Can you try `/* stylelint-disable-next-line max-nesting-depth no-descending-specificity */`? I'm not 100% sure `no-descending-specificity` is the right eslint rule but that's the one I've been seeing lately. If that doesn't work, maybe just leave another comment above it stating which lint errors you're overriding so it's easier to fix it later if we want.